elasticsearch/conf.sls

```diff
@@ -32,3 +33,15 @@
     - require_in:
       - service: elasticsearch
 {%- endif %}
+
+# Configure jvm.options
+/etc/elasticsearch/jvm.options:
+  file.append:
```

Comment:
If the file already exists you may want to use [file.blockreplace](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.blockreplace) with `append_if_not_found: True` so it will add a "managed section" to the file like so:


```
# START managed zone
stuff-here: 1
do-stuff: 232
# END managed zone
```

Otherwise this may end up appending stuff to a file multiple times if something manually changed and basically makes the section managed instead of the whole file. 

This works too though.